content/sensu-go/6.8/catalog/catalog-reference.md

```diff
@@ -0,0 +1,1265 @@
+---
+title: "Catalog integrations reference"
+linkTitle: "Catalog Integrations Reference"
+reference_title: "Catalog integrations"
+type: "reference"
+description: "Use Sensu's Catalog API to build a private catalog of Sensu integrations. Read the reference to create integrations for your catalog."
+weight: 80
+version: "6.8"
+product: "Sensu Go"
+menu: 
+  sensu-go-6.8:
+    parent: catalog
+---
+
+{{% notice commercial %}}
+**COMMERCIAL FEATURE**: Access the Sensu Catalog and integrations in the packaged Sensu Go distribution.
+For more information, read [Get started with commercial features](../../commercial/).
+{{% /notice %}}
+
+{{% notice note %}}
+**NOTE**: The Sensu Catalog is in public preview and is subject to change.
+{{% /notice %}}
```

Comment:
Is this still true? We have this note on our existing catalog page (https://docs.sensu.io/sensu-go/latest/web-ui/sensu-catalog/).